public/app/features/dashboard-scene/scene/LibraryVizPanel.tsx

```diff
@@ -78,29 +77,6 @@ export class LibraryVizPanel extends SceneObjectBase<LibraryVizPanelState> {
     };
 
     const panel = new VizPanel(vizPanelState);
-    const gridItem = this.parent;
-
-    if (libPanelModel.repeat && gridItem instanceof DashboardGridItem && gridItem.parent instanceof SceneGridLayout) {
-      this._parent = undefined;
```

Comment:
This is the wierd thing that i'm trying to get rid of. Plus the complexity of the repeater creation, and state updates below.